main.ts

```
@@ -434,10 +544,110 @@ class UnearthedSettingTab extends PluginSettingTab {
+					await this.plugin.saveSettings();
+					this.display();
+				})
+			)
+			.addTextArea(
+				(textArea) =>
+					(textArea
+						.setValue(this.plugin.settings.sourceTemplate)
+						.onChange(async (value) => {
+							this.plugin.settings.sourceTemplate = value;
+							await this.plugin.saveSettings();
+						}).inputEl.style.height = "200px")
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)